task1.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Task1</title>
+        <link rel="stylesheet" type="text/css" href="main.css">
+    </head>
+    <body>
+        <header>
+            <div class="logo">
+                <h1>WEBSITE NAME</h1>
+                <p>lorem ipsum dolor sit amet</p>
+            </div>
+            <nav>
+                <a href="#">Home</a>
```

Comment:
![image](https://user-images.githubusercontent.com/2150198/89173611-34bb0c00-d58d-11ea-85d4-1f1178ecf4a5.png)
button is cut off in mobile view